WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -19,7 +20,42 @@ def new
   end
 
   def index
-    @competitions = Competition.all.select([:id, :name, :cityName, :countryId]).order(:year, :month, :day).reverse_order
+    @regions = { '' => [["All","all"]],
```

Comment:
I checked, it is possible to generate a standalone `<option>` outside of a `<optgroup>`, which I believe would look much nicer than what we have right now. See:

```ruby
~/gitting/worldcubeassociation.org/WcaOnRails @kaladin> bin/rails console
Loading development environment (Rails 4.2.1)
irb(main):001:0> include ActionView::Helpers::FormOptionsHelper
=> Object
irb(main):002:0> p content_tag(:option, "All", value: "foo") + grouped_options_for_select([['group1', ['a', 'b']]])
"<option value=\"foo\">All</option><optgroup label=\"group1\"><option value=\"a\">a</option>\n<option value=\"b\">b</option></optgroup>"
=> "<option value=\"foo\">All</option><optgroup label=\"group1\"><option value=\"a\">a</option>\n<option value=\"b\">b</option></optgroup>"
irb(main):003:0> 
```